### DISCUSSION TITLE
Discord registration, Driver vs BaseCase vs SB

### DISCUSSION THREAD

**Discussion by spacejlyfish** [seleniumbase/SeleniumBase#2752] [Q&A] [answered]
i am testing a script using the Driver manager as direct import with discord registration. 
It seems to get detected most of the time as discords requiring phone verification instead of email.
I'm not sure if the detection comes from the webdriver or the residential proxies or use of self-hosted address or any other factores
but i does seem to have a higher detection rate when using driver.type :

self.driver = Driver(uc=True, proxy=self.proxy, headless=headless)
self.driver.set_window_position(-1920,0, windowHandle='current')
self.driver.maximize_window()
self.driver.get("https://discord.com/register")
wait = round(random.uniform(min_val, max_val), decimal)
self.driver.sleep(wait)
self.driver.type('input[name="email"]', email, timeout=20)

i have also tried with 
self.driver.uc_open_with_reconnect("https://discord.com/register", 5.5) but still gets detected.

Would trying SB context  or BaseCase syntax worth trying?
Anyone succeded avoiding detection on discord?

**Comment by mdmintz** [comment, marked as answer]
This worked for me: (Loaded the page normally)

```python
from seleniumbase import SB

with SB(uc=True, test=True, incognito=True) as sb:
    url = "https://discord.com/register"
    sb.driver.uc_open_with_reconnect(url, 5)

    breakpoint()
```

**Reply by spacejlyfish** [reply]
My issue is not loading the page, but when i get to the end of the registration process, where it should require email verif, it asks phone verif. this happens if something during the process indicates automation. And i believe it happens withing those first few lines.

i though maybe those lines could be the issue
self.driver.set_window_position(-1920,0, windowHandle='current')
self.driver.maximize_window()

i'm running out of ideas

**Reply by mdmintz** [reply]
Never maximize the window or change window position. Keep the scripts as minimal as possible. Use `uc_click(selector)` whenever clicking. Some sites are better protected than others. SeleniumBase provides the tools, but ultimately it's up to people to create their own scripts.

**Reply by spacejlyfish** [reply]
Thanks i have tried it but still gets detected.
i only uses uc_click, type and sleep.

**Reply by mdmintz** [reply]
Should work fine for regular login. It doesn't do registrations, though. Different protections.
